libr/core/cmd_anal.c

```diff
@@ -6999,22 +6999,22 @@ static int cmd_anal_all(RCore *core, const char *input) {
 	case '?': r_core_cmd_help (core, help_msg_aa); break;
 	case 'b': // "aab"
 		cmd_anal_blocks (core, input + 1);
-		break; // "aab"
+		break;
 	case 'f': // "aaf"
-		{
-			int analHasnext = r_config_get_i (core->config, "anal.hasnext");
-			r_config_set_i (core->config, "anal.hasnext", true);
-			r_core_cmd0 (core, "afr@@c:isq");
-			r_config_set_i (core->config, "anal.hasnext", analHasnext);
-		}
+	{
```

Comment:
Not sure if i prefer the indented braces or not..

Reply:
I think not indented is better. Otherwise you have 2 indentations for almost nothing. That said, the blocks shouldn't even be needed in a `switch-case`, but radare2 source code is full of super-messy switch/cases... :(

And, again, I'm happy as long as a tool automatically do that :D